aeroflexSup/aeroflex_common.db

```diff
@@ -0,0 +1,231 @@
+}
+
+record(mbbi, "$(P)CARRIER_FREQ_UNITS:SP")
+{
+    field(DESC, "Set Carrier Frequency Units")
+	field(UDFS, "NO_ALARM")
+	info(autosaveFields, "VAL")
+	
+	field(ZRST, "HZ")
+	field(ONST, "kHZ")
+	field(TWST, "MHZ")
+	field(THST, "GHZ")
+}
+
+record(ao, "$(P)CAR_FREQ_CONV")
+{
+    field(DESC, "Carr freq val converted to units")
+	field(EGU, "")
+	field(PREC, "2")
+}
+
+record(calcout, "$(P)CALC_RETURN_CAR_FREQ")
+{
+	field(DESC, "Convert car freq to kHZ,MHZ,GHZ")
+	field(INPA, "$(P)CARRIER_FREQ")
+	field(INPB, "$(P)CAR_FREQ_READOUT_UNITS")
+	field(CALC, "A / 10 ** (3 * B)")
+	field(OUT, "$(P)CAR_FREQ_CONV PP")
+}
+
+record(calcout, "$(P)SEND_CAR_FREQ_PARAMS")
+{
+    field(DESC, "Set carrier frequency button")
+    field(SCAN, "Passive")
+	field(INPA, "$(P)CARRIER_FREQ:SP_NO_ACTION")
+	field(CALC, "A")
+	field(OUT, "$(P)CARRIER_FREQ:SP PP")
+}
+
+record(scalcout, "CAR_FREQ_CONV_UNIT") {
+    field(DESC, "Set car freq conv unit")
+	field(SCAN, "1 second")				
+	field(INPA, "$(P)CAR_FREQ_READOUT_UNITS")
+    field(CALC, "(A=0)?'HZ':(A=1)?'kHZ':(A=2)?'MHZ':(A=3)?'GHZ':''")
+    field(OUT, "$(P)CAR_FREQ_CONV.EGU PP")
+}
+
+record(ai, "$(P)RF_LEVEL")
+{
+    field(SCAN, "2 second")
+    field(DESC, "RF level")
+    field(DTYP, "stream")
+	field(EGU, "DBM")
+    field(INP, "@aeroflex_common.proto getRFLevel $(PORT)")
+	field(PREC, "$(RF_PREC)")
+	info(archive, "VAL")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:RF_LEVEL")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)RF_LEVEL:SP")
+{
+    field(DESC, "Set RF level")
+	field(DTYP, "stream")
+	field(OUT, "@aeroflex_common.proto setRFLevel $(PORT)")
+	field(PREC, "$(RF_PREC)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:RF_LEVEL:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)RF_LEVEL:SP_NO_ACTION")
+{
+	field(PREC, "$(RF_PREC)")
+	info(autosaveFields, "VAL")
+	field(UDFS, "NO_ALARM")
+}
+
+record(dfanout, "$(P)SEND_RF_LVL_PARAMS")
+{
+    field(DESC, "Set RF level button")
+    field(SCAN, "Passive")
+    field(OMSL, "closed_loop")
+    field(DOL, "$(P)RF_LEVEL:SP_NO_ACTION")
+    field(OUTA, "$(P)RF_LEVEL:SP PP")
+}
+
+record(mbbi, "$(P)MODE:SP_NO_ACTION")
+{
+	info(autosaveFields, "VAL")
+	field(UDFS, "NO_ALARM")
+	
+	field(ZRST, "AM,FM")
+	field(ONST, "AM,PM")
+	field(TWST, "AM")
+	field(THST, "FM")
+	field(FRST, "PM")
+}
+
+record(bo, "$(P)PULSE_CHECK:SP")
+{
+    field(DESC, "Enables pulsed modulation")
+    field(SCAN, "Passive")
+	field(UDFS, "NO_ALARM")
+	
+	info(autosaveFields, "VAL")
+    
+    field(ZNAM, "Pulse disabled")
+    field(ONAM, "Pulse enabled")
+    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PULSE_CHECK:SP")
+}
+
+record(stringin, "$(P)ERROR")
+{
+    field(SCAN, "2 second")
+    field(DESC, "Error")
+    field(DTYP, "stream")
+
+    field(INP, "@aeroflex_common.proto getError $(PORT)")
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:ERROR")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(stringout, "$(P)DEV_TYPE")
```

Comment:
```suggestion
record(stringin, "$(P)DEV_TYPE")
```
Does this need to be a `stringout`? We just set `VAL` to the device type macro, right?